Source/WebCore/page/scrolling/ScrollingTreeFrameHostingNode.cpp

```diff
@@ -73,7 +73,7 @@ void ScrollingTreeFrameHostingNode::removeHostedChildren()
 {
     auto hostedChildren = std::exchange(m_hostedChildren, { });
     for (auto& children : hostedChildren)
-        scrollingTree().removeNode(children->scrollingNodeID());
+        scrollingTree().removeNode(children->scrollingNodeID(), nullptr);
```

Comment:
Just provide a default argument to `removeNode()`

Reply:
Fixed.